manifests/operators/prometheus-bosh-exporter.yml

```diff
@@ -0,0 +1,65 @@
+# Prometheus Scrape Config
+- type: replace
+  path: /instance_groups/name=prometheus/jobs/name=prometheus/properties/prometheus/scrape_configs/-
+  value: 
+    job_name: bosh
+    static_configs:
+    - targets: 
+      - localhost:9190 # bosh_exporter
+
+# Exporter jobs
+- type: replace
+  path: /instance_groups/name=prometheus/jobs/-
+  value: 
+    name: bosh_exporter
+    release: prometheus
+    properties:
+      bosh_exporter:
+        metrics:
+          environment: ((metrics_environment))
+        bosh:
+          url: ((bosh_url))
+          username: ((bosh_username))
+          password: ((bosh_password))
+          ca_cert: ((bosh_ca_cert)
+
+# Grafana Dashboards
+- type: replace
+  path: /instance_groups/name=grafana/jobs/-
+  value: 
+    name: system_dashboards
+    release: prometheus
+
+- type: replace
+  path: /instance_groups/name=grafana/jobs/name=grafana/properties/grafana/prometheus/dashboard_files/-
+  value: /var/vcap/packages/system_dashboards/*.json
+
+- type: replace
+  path: /instance_groups/name=grafana/jobs/-
+  value: 
+    name: bosh_dashboards
+    release: prometheus
+
+- type: replace
+  path: /instance_groups/name=grafana/jobs/name=grafana/properties/grafana/prometheus/dashboard_files/-
+  value: /var/vcap/packages/bosh_dashboards/*.json
+
+# Prometheus Alerts
+- type: replace
+  path: /instance_groups/name=prometheus/jobs/-
+  value: 
+    name: bosh_alerts
+    release: prometheus
+
+- type: replace
+  path: /instance_groups/name=prometheus/jobs/name=prometheus/properties/prometheus/rule_files/-
+  value: /var/vcap/packages/bosh_alerts/*.alerts
+
+# Alertmanager Grouping
```

Comment:
This allowed bosh-exporter and boshtsdb-exporter alerts to be grouped. As can be seen in the following payload (from alertmanager webhook) https://gist.github.com/rkoster/80509fc149a88d9ab350cf2b2d5092b7#file-grouped_alerts-json-L126-L129